install.sh

```diff
@@ -125,17 +125,39 @@ echo Using SDK version $SDKVER
 # The original casing of file names is preserved though, by adding lowercase
 # symlinks instead of doing a plain rename, so files can be referred to with
 # either the out of the box filename or with the lowercase name.
-$ORIG/lowercase -map_winsdk -symlink kits/10/include/$SDKVER/um
-$ORIG/lowercase -map_winsdk -symlink kits/10/include/$SDKVER/shared
-$ORIG/lowercase -map_winsdk -symlink kits/10/include/$SDKVER/winrt
-$ORIG/fixinclude -map_winsdk kits/10/include/$SDKVER/um
-$ORIG/fixinclude -map_winsdk kits/10/include/$SDKVER/shared
-$ORIG/fixinclude -map_winsdk kits/10/include/$SDKVER/winrt
+for incdir in um shared winrt km; do
+    SDK_INCDIR="kits/10/include/$SDKVER/$incdir"
+
+    if [ -d "$SDK_INCDIR" ]; then
+        $ORIG/lowercase -map_winsdk -symlink "$SDK_INCDIR"
+    fi
+done
+
+for incdir in um shared winrt km; do
+    SDK_INCDIR="kits/10/include/$SDKVER/$incdir"
+
+    if [ -d "$SDK_INCDIR" ]; then
+        $ORIG/fixinclude -map_winsdk "$SDK_INCDIR"
```

Comment:
Can you merge these two loops? It would deviate from the order in the script originally, but I think it should be harmless (fixinclude doesn't look at files outside of the given directory).

Reply:
I was not sure whether there are dependencies between `lowercase` and `fixinclude`, and just assumed that all headers must be lowercased for `fixinclude` to work. I'll merge them into one loop, then.